Makefile

```diff
@@ -1,28 +1,58 @@
-.PHONY: dockerbuild dockerpush test testonce ruff pylint black lint
+.PHONY: dockerbuild dockerpush test testonce ruff black lint isort pre-commit-check requirements-update requirements setup
 VERSION ?= latest
 IMAGENAME = eodhp-utils
-DOCKERREPO ?= 312280911266.dkr.ecr.eu-west-2.amazonaws.com
+DOCKERREPO ?= public.ecr.aws/n1b3o1k2/ukeodhp
 
-dockerbuild:
-	DOCKER_BUILDKIT=1 docker build -t ${IMAGENAME}:${VERSION} .
-
-dockerpush: dockerbuild testdocker
-	docker tag ${IMAGENAME}:${VERSION} ${}DOCKERREPO}/${IMAGENAME}:${VERSION}
-	docker push ${DOCKERREPO}/${IMAGENAME}:${VERSION}
+# dockerbuild:
+# 	DOCKER_BUILDKIT=1 docker build -t ${IMAGENAME}:${VERSION} .
+#
+# dockerpush: dockerbuild testdocker
+# 	docker tag ${IMAGENAME}:${VERSION} ${DOCKERREPO}/${IMAGENAME}:${VERSION}
+# 	docker push ${DOCKERREPO}/${IMAGENAME}:${VERSION}
```

Comment:
Do we want these commented out or can we delete?

Reply:
Removed